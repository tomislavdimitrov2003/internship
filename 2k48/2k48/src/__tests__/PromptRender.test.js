import Prompt from "../PromptRender";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import React from 'react';

it('renders propmt correctly', () => {
    const wrapper = shallow(<Prompt finalScore={1000}/>);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
});