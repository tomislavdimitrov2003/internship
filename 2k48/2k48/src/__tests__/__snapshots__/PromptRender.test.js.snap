// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders propmt correctly 1`] = `
<div>
  <WithStyles(ForwardRef(Dialog))
    onClose={false}
    open={false}
  >
    <WithStyles(ForwardRef(DialogTitle))>
      Game Over
    </WithStyles(ForwardRef(DialogTitle))>
    <WithStyles(ForwardRef(DialogContent))>
      <WithStyles(ForwardRef(DialogContentText))>
        You Finished with score of 
        1000
        !
      </WithStyles(ForwardRef(DialogContentText))>
      <WithStyles(ForwardRef(TextField))
        autoFocus={true}
        fullWidth={true}
        id="username"
        label="Username"
        margin="dense"
        type="username"
        variant="standard"
      />
    </WithStyles(ForwardRef(DialogContent))>
    <WithStyles(ForwardRef(DialogActions))>
      <WithStyles(ForwardRef(Button))
        onClick={[Function]}
      >
        Submit
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(DialogActions))>
  </WithStyles(ForwardRef(Dialog))>
</div>
`;
